src/main/java/uk/gov/hmcts/cmc/claimstore/config/ServiceTokenGeneratorConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.cmc.claimstore.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cloud.netflix.feign.EnableFeignClients;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Lazy;
+import uk.gov.hmcts.reform.authorisation.ServiceAuthorisationApi;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.authorisation.generators.CachedServiceAuthTokenGenerator;
+import uk.gov.hmcts.reform.authorisation.generators.ServiceAuthTokenGenerator;
+
+@Configuration
+@Lazy
+@EnableFeignClients(basePackageClasses = ServiceAuthorisationApi.class)
```

Comment:
Is this needed? it is being enabled in the application class as well

Reply:
Not required. removed.